docs/reference/licenses.nuget.org.md

```diff
@@ -0,0 +1,69 @@
+# licenses.nuget.org
+
+## Rationale
+
+With introduction of the [license expressions](nuspec.md#license) a requirement emerged to have a reliable service
+that would provide a reference text for individual license identifiers, exception identifiers or license expressions.
+An additional requirement for this service is to have a stable URL schema, that is not susceptible to the link rot,
+so that we can safely use it to provide backwards compatibility for older clients.
+
+Licenses.nuget.org fulfills that role. Gallery uses it to provide the license text reference for packages that
+specify their license using a license expression. `nuget.exe pack` operation sets the [`licenseUrl`](nuspec.md#licenseurl)
+element to point to licenses.nuget.org to provide backwards compatibility with older clients that don't support
+the `license` element.
+
+## Protocol
+
+Licenses.nuget.org is intended to be viewed by people in their browsers, no machine-readable responses are provided.
+HTTPS protocol must be used and requests constructed in a certain way. It only supports `GET` requests. It accepts
+license expressions or license exception identifiers as an input in a way specified below.
+
+### License expressions
+
+#### Request
+
+License expressions (including the trivial cases when expression consists of a single license) have to be
```

Comment:
Are the URLs case insensitive? It may be worth pointing this out.

Reply:
License expressions are case sensitive, and service requires valid license expressions to be supplied.

Reply:
Is this mentioned somewhere? I don't see it...

Reply:
The page links to [`license`](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1281#license) element documentation, that in turn points to SPDX specification, which defines the exact syntax for license expressions. So there is no explicit mention of case sensitivity until the syntax BNF, so I guess can add a note stating that.

Regarding the list of accepted licenses, there is no such thing neither for Gallery, nor for this service as of now.